newIDE/app/src/Utils/Analytics/EventSender.js

```diff
@@ -136,7 +137,36 @@ export const identifyUserForAnalytics = (
     const userId = firebaseUser ? firebaseUser.uid : getUserUUID();
     posthog.identify(userId, userProperties);
     posthogLastPropertiesSent = stringifiedUserProperties;
+    userIdentified = true;
+  }
+};
+
+/**
+ * Use this function on signup to link the Firebase ID to the UUID.
+ * This is important to do this alias only Once per ID, so only on signup.
```

Comment:
It's not clear why it's important.
I would rather explain "This is only done on sign-up because ..."